### PR TITLE
fix(ui): ui changes for #9695

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked UI changes for #9695 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

https://user-images.githubusercontent.com/51777795/213231727-7b5c2184-9d23-41d0-87bb-ad47f64cb8fc.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ const ExploreQuickFilters: FC<ExploreQuickFiltersProps> = ({
             const uniqueOptions = [
               ...new Set(
                 suggestOptions.map((op) => ({
-                  key: op._source.name,
+                  key: op._source.displayName ?? op._source.name,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx:112]: @aniketkatkar97 , Do we want displayName as the key?

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ describe('Test Explore component', () => {
     );
     const searchData = await findByTestId(container, 'search-data');
     const wrappedContent = await findByTestId(container, 'wrapped-content');
-    const tabs = await findAllByTestId(container, /tab/i);
+    const tabs = await findAllByTestId(container, /-tab/i);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx:82]: @aniketkatkar97 , Why we are changing this?

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx:112]: Yes we will be passing displayName to the API so it is necessary

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx:82]: So as I changed datatestid for search-dropdowns changed from 'search-dropdown' to 'search-dropdown-${dropdownName}', `/tab/i` was also returning the div for database dropdown while fetching tabs divs.

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx:82]: @aniketkatkar97, Please add comments for such changes.

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js [hunk 3]
```diff
@@ -175,9 +175,6 @@ describe('Advance search should work properly for Add Group functionality', () =
     it(`Verify Add group functionality for All with ${operator.name} operator & condition ${CONDITIONS_MUST.contains.name} and ${CONDITIONS_MUST_NOT.notContains.name} `, () => {
       Object.values(FIELDS).forEach((field) => {
         let val = field.searchCriteriaSecondGroup;
-        if (field.owner) {
-          val = field.searchCriteriaSecondGroup.split(' ')[0];
-        }
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js:180]: why we are removing this?

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js:180]: earlier we were checking if only name of the user is present in the API call. Example for 'Aaron Johnson' we were asserting only 'Aaron'. I removed that ant introduced encodeURL to check the whole thing.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**